frontend/src/components/PlannerCart/styles.js

```diff
@@ -0,0 +1,70 @@
+import { Button as Btn } from "antd";
+import styled from "styled-components";
+
+const plannerCartRoot = styled.div`
```

Comment:
Could we capitalise the first letter for the styled components? i.e. `const PlannerCartRoot`